trace_decoder/Cargo.toml

```
@@ -10,6 +10,8 @@ homepage.workspace = true
 keywords.workspace = true
 
 [dependencies]
+alloy.workspace = true
```

nit: can we use the same formatting as all other dependencies?
```suggestion
alloy = { workspace = true }
```

---

This is the default for `cargo add`. I have to manually edit it every time this comes up - I propose we make this this norm, but I'm just going to edit these to agree with your suggestion for now.

---

I'm okay with `foo.workspace = true` everywhere FWIW